templates.go

```diff
@@ -1,68 +1,48 @@
 package egoscale
 
-// Template represents a machine to be deployed
-//
-// See: http://docs.cloudstack.apache.org/projects/cloudstack-administration/en/latest/templates.html
-type Template struct {
-	Account               string            `json:"account,omitempty"`
-	AccountID             string            `json:"accountid,omitempty"`
-	Bootable              bool              `json:"bootable,omitempty"`
-	Checksum              string            `json:"checksum,omitempty"`
-	Created               string            `json:"created,omitempty"`
-	CrossZones            bool              `json:"crossZones,omitempty"`
-	Details               map[string]string `json:"details,omitempty"`
-	DisplayText           string            `json:"displaytext,omitempty"`
-	Domain                string            `json:"domain,omitempty"`
-	DomainID              string            `json:"domainid,omitempty"`
-	Format                string            `json:"format,omitempty"`
-	HostID                string            `json:"hostid,omitempty"`
-	HostName              string            `json:"hostname,omitempty"`
-	Hypervisor            string            `json:"hypervisor,omitempty"`
-	ID                    string            `json:"id,omitempty"`
-	IsDynamicallyScalable bool              `json:"isdynamicallyscalable,omitempty"`
-	IsExtractable         bool              `json:"isextractable,omitempty"`
-	IsFeatured            bool              `json:"isfeatured,omitempty"`
-	IsPublic              bool              `json:"ispublic,omitempty"`
-	IsReady               bool              `json:"isready,omitempty"`
-	Name                  string            `json:"name,omitempty"`
-	OsTypeID              string            `json:"ostypeid,omitempty"`
-	OsTypeName            string            `json:"ostypename,omitempty"`
-	PasswordEnabled       bool              `json:"passwordenabled,omitempty"`
-	Project               string            `json:"project,omitempty"`
-	ProjectID             string            `json:"projectid,omitempty"`
-	Removed               string            `json:"removed,omitempty"`
-	Size                  int64             `json:"size,omitempty"`
-	SourceTemplateID      string            `json:"sourcetemplateid,omitempty"`
-	SSHKeyEnabled         bool              `json:"sshkeyenabled,omitempty"`
-	Status                string            `json:"status,omitempty"`
-	Zoneid                string            `json:"zoneid,omitempty"`
-	Zonename              string            `json:"zonename,omitempty"`
+// ListRequest builds the ListTemplates request
+//TODO Tags
+func (temp *Template) ListRequest() (ListCommand, error) {
+	req := &ListTemplates{
```

Comment:
move the TODO within the func, so the `godoc` doesn't show it.

![screenshot from 2018-04-11 11-08-24](https://user-images.githubusercontent.com/1388/38607367-b3ab7c7a-3d78-11e8-94a5-c90bcaa7c091.png)